### PR TITLE
Create Mike1.json

### PR DESCRIPTION
**Pull request by angel1967**
nuevo update

### CODE REVIEW

**Comment by davidnhz** [comment]
👍

File: _pins/Mike1.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: angel1967
+latitude: 20.650945
+Longitude: -103.403149
```
Comment (davidnhz) [inline _pins/Mike1.json]: Longitude must me uncapitalized like this `longitude`

Comment (crichID) [inline _pins/Mike1.json]: @angel1967 if you make this one change, you should be able to merge your file! If you need help making changes, check out this quick video we made on how to correct common issues in this activity: https://vimeo.com/201011288

**Comment by brianamarie** [comment]
@angel1967  :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

**Pull request merged**